dfc/tests/smoke_test.go

```diff
@@ -218,7 +218,9 @@ func fillWithRandomData(seed int64, fileSize uint64, objList []string, bucket st
 			}
 			errch <- err
 		}
-		filesput <- fname
+		if filesput != nil {
```

Comment:
are there any places that calls this with files put == nil? if not, no need to check, it will crash if a nil is passed here and coder will find that out right away, there is no runtime possible calling this with ch == nil